src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -121,26 +121,25 @@ static class LeaderboardView<TEntry> where TEntry : WebApi.IEntry
                 ColorConsole.WriteLine();
                 foreach (var line in lines)
                 {
-                    ColorConsole.WriteLine(new ColorToken[] { padding }.Concat(line).ToArray());
+                    ColorConsole.WriteLine(new ColorToken[]
+                    {
+                        padding
+                    }.Concat(line).ToArray());
                 }
 
                 ColorConsole.WriteLine(
                     padding,
-                    $"Total {itemsName}:".Gray(),
-                    " ",
-                    $"{currentLeaderboard?.Count ?? 0:N0}"
-                        .Color(currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0]),
-                    $" since ".Gray(),
-                    $"{currentLeaderboard.Since?.ToLocalTime().ToString() ?? ("started")}".Gray());
+                    $"{currentLeaderboard.Since?.ToLocalTime().ToString() ?? ("started")} to {currentLeaderboard.LastActivityDate?.ToLocalTime().ToString() ?? ("started")}".Gray());
 
                 var maxMessageLength = 0;
                 var refreshTime = DateTime.UtcNow.AddMilliseconds(refreshInterval);
                 using (var timer = new Timer(c =>
                 {
-                    var timeLeft = new TimeSpan(0, 0, 0, (int)Math.Round((refreshTime - DateTime.UtcNow).TotalSeconds));
-                    var message = $"\r{padding}{DateTime.UtcNow.ToLocalTime()} · Refreshing in {timeLeft.Humanize()}...";
-                    maxMessageLength = Math.Max(maxMessageLength, message.Length);
-                    ColorConsole.Write(message.PadRight(maxMessageLength).DarkGray());
+                    var timeLeft = new TimeSpan(0, 0, 0, (int) Math.Round((refreshTime - DateTime.UtcNow).TotalSeconds));
+                    var totalMessage = $"Total {itemsName}: {currentLeaderboard?.Count ?? 0:N0}"; 
+                    var refreshingMessage = $" · Refreshing in {timeLeft.Humanize()}...";
+                    maxMessageLength = Math.Max(maxMessageLength, totalMessage.Length + refreshingMessage.Length);
+                    ColorConsole.Write("\r", padding, totalMessage.Color(currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0]), refreshingMessage.PadRight(maxMessageLength).DarkGray());

```

Comment:
This is padding too much since it's padding the partial `refreshingMessage` to the length of `totalMessage`.

Also, we don't want to colour the entire message with the movement colour when the count increases. Only `currentLeaderboard?.Count ?? 0`. The rest of the text should remain dark grey.